masonite/session.py

```diff
@@ -0,0 +1,47 @@
+
+
+class Session():
+    """
+    Class Session for manage sessions of the app
+    """
+    _session = {}
+
+    def __init__(self, environ):
+        """
+        Constructor
+        """
+        self.environ = environ
+
+    def get(self, key):
+        """
+        Get a session from object _session
+        """
+        ip = self.__get_client_address()
+        try:
+            session = self._session[ip][key]
+        except KeyError:
+            session = ''
```

Comment:
Let's make this None instead. It makes sense that if you try to get something and it doesn't exist then it should return None. If it returns an empty string that seems odd to me

I think this should actually just look like:

```python
ip = self.__get_client_address()

try:
    return self._session[ip][key]
except KeyError:
    return None
```

just to shorten it up. but do some testing on that, that might not work. Doing that should make the same tests pass but you just have to check for `None` instead of a blank string `''`